src/shared/bus-wait-for-jobs.h

```diff
@@ -5,12 +5,17 @@
 
 #include "macro.h"
 
+typedef enum WaitJobsFlags {
+        BUS_WAIT_JOBS_LOG_ERROR   = 1 << 0,
+        BUS_WAIT_JOBS_LOG_SUCCESS = 1 << 1,
```

Comment:
The `BUS_WAIT_JOBS_LOG_SUCCESS` flag is not used in the first commit. Please introduce it in the second commit.